src/lib/flexbox/api/flex.ts

```diff
@@ -301,8 +311,21 @@ export class FlexDirective extends BaseFxDirective implements OnInit, OnChanges,
     let usingCalc = String(basis).indexOf('calc') > -1;
     let isPx = String(basis).indexOf('px') > -1 || usingCalc;
 
-    css[min] = (basis == '0%') ? 0 : isPx ? basis : null;
-    css[max] = (basis == '0%') ? 0 : usingCalc ? null : basis;
```

Comment:
Sure, as long we get to a place where #153 and things stemming from it are resolved. I really done care much about the minor details..

Thanks for all the feadback!